packages/scandipwa/src/component/CheckoutDeliveryOptions/CheckoutDeliveryOptions.component.js

```diff
@@ -86,13 +92,28 @@ export class CheckoutDeliveryOptions extends PureComponent {
     }
 
     render() {
+        const {
+            estimateAddress,
+            shippingMethods,
+            onStoreSelect,
+            onShippingMethodSelect,
+            setSelectedShippingMethodCode
+        } = this.props;
+
         return (
             <div block="CheckoutDeliveryOptions">
                 { this.renderHeading() }
                 <ul block="CheckoutPayments" elem="Methods">
                     { this.renderShippingMethods() }
                 </ul>
                 { this.renderSelectedShippingMethod() }
+                <StoreInPickUp
+                  estimateAddress={ estimateAddress }
+                  shippingMethods={ shippingMethods }
+                  onStoreSelect={ onStoreSelect }
+                  onShippingMethodSelect={ onShippingMethodSelect }
+                  setSelectedShippingMethodCode={ setSelectedShippingMethodCode }
+                />
```

Comment:
Is really always neccessary, cannot it just be there when needed? I mean in `renderSelectedShippingMethod` ?